bin/compile

```diff
@@ -429,8 +429,12 @@ status "Installing platform packages..."
 export export_file_path=$bp_dir/export
 export profile_dir_path=$build_dir/.profile.d
 export providedextensionslog_file_path=$(mktemp -t "provided-extensions.log.XXXXXX" -u)
-# grep and sed in a subshell with || true, or a failing grep match will cause the exit code to be 1 if "composer install" fails with code 2
-if composer install -d "$build_dir/.heroku/php" ${HEROKU_PHP_INSTALL_DEV-"--no-dev"} 2>&1 | tee $build_dir/.heroku/php/install.log | ( grep --line-buffered -E '^  - (Instal|Enab)ling heroku-sys/' | sed -u -E -e 's/^  - (Instal|Enab)ling /- /' -e 's/heroku-sys\///g' || true ) | indent; then
+# we make a new file descriptor for the installer plugin to log "human readable" display output to, duplicated to stdout (via indent)
+exec {PHP_PLATFORM_INSTALLER_DISPLAY_OUTPUT_FDNO}> >(indent)
+# the installer picks up the FD number in this env var, and writes a "display output" log to it
+# meanwhile, the "raw" output of 'composer install' goes to install.log in case we need it later
+export PHP_PLATFORM_INSTALLER_DISPLAY_OUTPUT_FDNO
+if composer install -d "$build_dir/.heroku/php" ${HEROKU_PHP_INSTALL_DEV-"--no-dev"} > $build_dir/.heroku/php/install.log 2>&1; then
```

Comment:
(optional; more for future reference) All of the Bash versions on our supported stacks have supported the `&>` syntax for a while

Reply:
Yea true, I need to do a bunch of quoting and shell style cleanup soon anyway